docs/pages/release_notes.md

```diff
@@ -1,5 +1,63 @@
 Release Notes {#releasenotes}
 ==============
+v3.4.0
+==============
+The source code with submodules can be downloaded directly from the following link:
+http://arrayfire.com/arrayfire_source/arrayfire-full-3.4.0.tar.bz2
```

Comment:
These lines are part of github release notes but not our documentation. You can remove these.